internal/sfio/migrate.go

```diff
@@ -262,6 +282,49 @@ func (f *ExperimentMigrationFilter) migrateMetricsV1alpha1(node *yaml.RNode) (*y
 	return node, nil
 }
 
+func (f *ExperimentMigrationFilter) appLabels(node *yaml.RNode) (labelApplication string, labelScenario string, err error) {
+	// Extract the metadata from the node
+	var md yaml.ResourceMeta
+	md, err = node.GetMeta()
+	if err != nil {
+		return
+	}
+
+	// Guess an application name using the experiment name
+	labelApplication = md.Labels[optimizeappsv1alpha1.LabelApplication]
+	if labelApplication == "" {
+		labelApplication = "default"
+
+		// Filter out buzzwords, and pure integers
+		var appName []string
+		drop := regexp.MustCompile("^example|experiment|final|yolo|[0-9]+$")
```

Comment:
❤️ 
Only filter that might be contentious would be stripping the numbers. I could imagine something like `myapp30` and `myapp40` if I were to try to associate version with the name

Reply:
No, that's a bug (the regexp is missing parens); it should let `myapp30` through, but it should take `myapp-30` down to `myapp`.